daccs_client/node.py

```diff
@@ -67,10 +83,6 @@ def last_updated(self) -> datetime:
     def daccs_version(self) -> str:
         return self._daccs_version
 
-    @property
```

Comment:
why delete this one?

Reply:
My thinking here was that the presence of both `status` and `is_online` function might lead to confusion, so I left this as an internal variable rather than make it directly available. What is your thought on this?

Reply:
Oh yeah, that makes sense. I wonder if we can still use the `_status` to indicate whether the last registry update for that node was successful of not. But maybe using the timestamp that shows the last update time is better for that.